Makefile

```diff
@@ -22,56 +18,67 @@ TEST_DEP = data/basic/basic.config \
 	bin/watch_cron_logs.pl \
 	bin/exception.py
 
+TEST_SKA = $(PWD)/test_ska
+
+# Define installation PREFIX as the sys.prefix of python in the PATH.
+PREFIX = $(shell python -c 'import sys; print(sys.prefix)')
```

Comment:
I like this idiom though I do wonder if $CONDA_PREFIX might be an option as well.

Reply:
This idiom is good enough.